bucket/foobar2000.json

```diff
@@ -28,22 +29,28 @@
             "    New-Item \"$persist_dir\\playlists\" -ItemType Directory -Force | Out-Null",
             "    if (Test-Path \"$env:Appdata\\foobar2000\") {",
             "        Write-Host -F yellow \"Copying old '$env:Appdata\\foobar2000' to '$persist_dir'\"",
-            "        Copy-Item \"$env:Appdata\\foobar2000\\*\" \"$dir\" -Recurse -Force",
+            "        Copy-Item \"$env:Appdata\\foobar2000\\*\" \"$dir\" -Force -Recurse",
+            "        if (Test-Path \"$env:Appdata\\foobar2000\\playlists-v*\") {",
+            "            Copy-Item \"$dir\\playlists-v*\" \"$persist_dir\\playlists\" -Force -Recurse",
+            "        }",
+            "        if (Test-Path \"$env:Appdata\\foobar2000\\user-components\") {",
+            "            Copy-Item \"$dir\\user-components\" \"$persist_dir\\user-components\" -Force -Recurse",
+            "        }",
             "    }",
             "}",
             "else {",
-            "    $shortVersion = $version.Split('.')[0, 1] -Join '.'",
-            "    Copy-Item \"$persist_dir\\playlists\" \"$dir\\playlist-v$shortVersion\" -Force -Recurse",
+            "    Copy-Item \"$persist_dir\\playlists\\*\" \"$dir\" -Force -Recurse",
             "}",
             "if (Test-Path \"$persist_dir\\theme.fth\") {",
             "    Copy-Item \"$persist_dir\\theme.fth\" \"$dir\" -Force",
             "}"
         ]
     },
+    "pre_install": "if(!(Test-Path \"$persist_dir\\user-components\")) { New-Item \"$dir\\user-components\" -ItemType Directory -Force | Out-Null}",
```

Comment:
Why??

You are using force, recurse in L37

Reply:
Having some weird behaviors with this line - will resolve shortly.

Reply:
Ah, was getting confused. Fixed.